### PR TITLE
Fix migration cloud to self-hosted

### PR DESCRIPTION
**Pull request by gewenyu99**
## What does this PR do?

@Haimantika This should fix https://github.com/appwrite/website/issues/231
## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: src/routes/docs/advanced/migrations/cloud/+page.markdoc [hunk 1]
```diff
@@ -19,6 +18,36 @@ Migrations makes it as easy as a couple clicks to move all of your self-hosted p
...
+{% info title="Keep in mind" %}
+Your self-hosted instance will generate an API Key in the background to pass to Appwrite Cloud. You can revoke this key after the migration process is complete.
+{% /info %}
+{% /section %}
+
+
+
+
+
+
```
Comment (stnguyen90) [inline src/routes/docs/advanced/migrations/cloud/+page.markdoc:50]: Stuff after needs to be deleted?

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Review by TGlide — APPROVED** [review]

File: src/routes/docs/advanced/migrations/cloud/+page.markdoc (hunk 1 above)
Comment (TGlide) [reply src/routes/docs/advanced/migrations/cloud/+page.markdoc]: @stnguyen90 I've removed them

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
See https://github.com/appwrite/website/pull/385#discussion_r1430651532

**Review by stnguyen90 — APPROVED** [review]
I'm not sure why the tests are failing as I built it myself and it worked out fine.

**Pull request merged**